client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -317,21 +369,29 @@ export const Admin = () => {
                 Add New Semester
               </button>
               <button
-                disabled={disableInit}
+                disabled={disableUpdateProfs}
```

Comment:
Referencing other comment -> can't disabled = {updating} ? unless there is a reason for having a separate disabled state for each button?